utils/chain-spec-builder/src/main.rs

```diff
@@ -191,6 +212,7 @@ impl ChainSpecBuilder {
     }
 }
 
```

Comment:
I noticed you disabled this lint at least twice. Do you mind to convert it to module-scope lint with a comment?